src/model/entity/enemy.go

```diff
@@ -24,10 +24,143 @@ const (
 	DirectionStop
 )
 
+type AttributeRate uint
+
+const (
+	AttributeRateLow      AttributeRate = 25
+	AttributeRateAverage  AttributeRate = 50
+	AttributeRateHigh     AttributeRate = 75
+	AttributeRateVeryHigh AttributeRate = 100
+)
+
+type HostilityRadius uint
+
+const (
+	HostilityRadiusLow     HostilityRadius = 2
+	HostilityRadiusAverage HostilityRadius = 4
+	HostilityRadiusHigh    HostilityRadius = 6
+)
+
 type Enemy struct {
-	BaseCharacter Character
-	Type          EnemyType
-	Hostility     uint
-	IsChasing     bool
-	Direction     Direction
+	Character       Character
+	Type            EnemyType
+	HostilityRadius HostilityRadius
+	IsChasing       bool
+	Direction       Direction
+}
+
+type Zombie struct {
+	Enemy Enemy
+}
+
+type Vampire struct {
+	Enemy          Enemy
+	HadFirstDamage bool
```

Comment:
Наверное, здесь можно было бы сделать счётчик "безусловных уклонений", чтобы: 
1. Проще было это понимать. Сейчас название переводится как "имел первый урон", что не совсем верно, т.к. уклоняется он от удара как такового, а не урона. То есть, условно говоря, если у нас удар не понесёт урон, то должен ли этот флаг перейти в `false`?
2. Проще с этим работать - легче увеличивать или уменьшать счётчик, чем выставлять/снимать флаги, как по мне.
3. Также, эта реализация будет более гибкой с точки зрения потенциальных игровых механик. Что если мы захотим (не в рамках задания, но допустим мы разработчики реальной игры) чтобы у нас вампир стал иметь больше, чем одно безусловное уклонение? Что если у нас появится "супер вампир", который будет иметь 3 уклонения и перезаряжать их каждые 20 секунд? С флагом нам придётся переписывать логику. Со счётчиком - ничего не поменяется.

По названию такое предложение:
```suggestion
	AbsoluteЕvasions uint
```